supabase/functions/cron_email/index.ts

```diff
@@ -0,0 +1,152 @@
+import { serve } from 'https://deno.land/std@0.199.0/http/server.ts'
+import { getEnv, methodJson, sendRes } from '../_utils/utils.ts'
+import type { BaseHeaders } from '../_utils/types.ts'
+import { trackEvent } from '../_utils/plunk.ts'
+import { supabaseAdmin } from '../_utils/supabase.ts'
+
+// This is required propoably due to this https://github.com/supabase/postgrest-js/issues/408
+interface AppWithUser {
+  app_id: string
+  name: string | null
+  user_id: {
+    email: string
+  }
+}
+
+const thresholds = {
+  updates: [
+    100,
+    1000,
+    10000,
+  ],
+  failRate: [
+    0.80,
+    0.90,
+    0.95,
+  ],
+  appOpen: [
+    500,
+    1500,
+    5000,
+  ],
```

Comment:
There's any variable name that we could use to know what these magic numbers are? I mean, is not clear if it's ms, seconds or any other thing
```suggestion
  updates: [
    100,
    1000,
    10000,
  ],
  failRate: [
    0.80,
    0.90,
    0.95,
  ],
  appOpen: [
    500,
    1500,
    5000,
  ],
```

Reply:
I will add comments

Reply:
Comments added [here](https://github.com/Cap-go/capgo/pull/359/commits/f3d138508065d6846e2790301eb663e1ec8ff16d)